api/proto/config/storage.proto

```diff
@@ -106,3 +106,48 @@ message Drive {
   // A value of 0 will indicate that no resizing is required
   int64 maxsizebytes = 10;
 }
+
+
+message ContentTree {
+  UUIDandVersion uuidandversion = 1;
```

Comment:
Can the version of ContentTree change? Can the Content Tree Change? Or should the Version always be 1?

Reply:
UUID is sufficient AFAIK

Reply:
changed it to uuid.